src/app/providers/datatables.provider.js

```diff
@@ -63,48 +63,87 @@ class DataTablesProvider {
                             <td colspan="${index.primary.length}">`;
 
         //--As a part of order online button--
+        const redirectUrl = 'https://test.smartservice.qld.gov.au/services/test/prodi/products';
```

Comment:
how does this work between dev and prd?

Reply:
I have amended that part by providing a new service called "EnvService" so that I can avoid hard-coding the endpoint. But I have not made the pull request with the latest code, yet, because there is something I have to confirm for something else.